database/java/2017/4995.yaml

```diff
@@ -0,0 +1,17 @@
+cve: 2017-4995
+title: "Jackson Configuration Allows Code Execution with Unknown Serialization Gadgets"
+description: >
+    When configured to enable default typing, Jackson contained a deserialization vulnerability that could lead to arbitrary code execution. Jackson fixed this vulnerability by blacklisting known "deserialization gadgets".
+references:
+    - http://www.cve.mitre.org/cgi-bin/cvename.cgi?name=2017-4995
+    - https://nvd.nist.gov/vuln/detail/CVE-2017-4995
+    - https://pivotal.io/security/cve-2017-4995
+
+affected:
+    - groupId: "org.springframework.security"
+      artifactId: "spring-security-web"
+      version:
+        - "<=4.2.2,4.2"
```

Comment:
`<=4.2.2` means that 4.1.x is also affected but it wasn't mentioned in the Pivotal's advisory and I don't see that `SecurityJackson2Modules` class is present in spring-security-{core,web}-4.1.x artifacts.